### PR TITLE
#171 feedback station installation

### PR DESCRIPTION
**Pull request by tidelbo**

### CODE REVIEW

**Review by JulienBardin — CHANGES_REQUESTED** [review]
Please add the .md extension to the content file.

File: _data/docs.yml [hunk 1]
```diff
@@ -90,6 +90,8 @@
       keywords : ["fridge monitoring","fridge"]
     - title: Leak Detector Installation Guide
       keywords : ["leak","water"]
+    - title: Feedback Station Installation
+      keywords : ["feedback station","facilities"]
```
Comment (JulienBardin) [inline _data/docs.yml:95]: It might be interesting to put the feedback before the "leak" so we keep an alphabetical order for specific installs

Comment (tidelbo) [reply _data/docs.yml:95]: IDK how to change the content file name

**Comment by JulienBardin** [comment]
Please add the .md extension to the content file.

**Pull request merged**